grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx

```diff
@@ -10,86 +10,71 @@ import { ApiTokenSettings } from 'containers/ApiTokenSettings/ApiTokenSettings';
-        </LegacyNavHeading>
+  return (
+    <div>
+      <LegacyNavHeading>
+        <Text.Title level={3} className={styles.title}>
+          Organization settings
+        </Text.Title>
+      </LegacyNavHeading>
 
-        <div className={cx('settings')}>
-          <Text.Title level={3} className={cx('title')}>
-            Resolution Note
-          </Text.Title>
-          <Field
-            loading={!currentOrganization}
-            label="Require a resolution note when resolving Alert Groups"
-            description={`Once user clicks "Resolve" for an Alert Group, they will be required to fill in a resolution note about the Alert Group`}
-          >
-            <WithPermissionControlTooltip userAction={UserActions.OtherSettingsWrite}>
-              <Switch
-                value={currentOrganization?.is_resolution_note_required}
-                onChange={(event) => {
-                  organizationStore.saveCurrentOrganization({
-                    is_resolution_note_required: event.currentTarget.checked,
-                  });
-                }}
-              />
-            </WithPermissionControlTooltip>
-          </Field>
-        </div>
-        {!isTopNavbar() && (
-          <div style={{ marginBottom: '20px' }}>
-            <Text.Title level={3} className={cx('title')}>
-              Teams and Access Settings
-            </Text.Title>
-            <TeamsSettings />
-          </div>
-        )}
-        <Text.Title level={3} className={cx('title')}>
-          API URL
+      <div className={styles.settings}>
+        <Text.Title level={3} className={styles.title}>
+          Resolution Note
         </Text.Title>
-        <div>
-          <Field>
-            <Input value={apiUrl} disabled />
-          </Field>
+        <Field
+          loading={!currentOrganization}
+          label="Require a resolution note when resolving Alert Groups"
+          description={`Once user clicks "Resolve" for an Alert Group, they will be required to fill in a resolution note about the Alert Group`}
+        >
+          <WithPermissionControlTooltip userAction={UserActions.OtherSettingsWrite}>
+            <Switch
+              value={currentOrganization?.is_resolution_note_required}
+              onChange={(event) => {
+                saveCurrentOrganization({
+                  is_resolution_note_required: event.currentTarget.checked,
+                });
+              }}
+            />
+          </WithPermissionControlTooltip>
+        </Field>
+      </div>
+      {!isTopNavbar() && (
+        <div style={{ marginBottom: '20px' }}>
+          <Text.Title level={3} className={styles.title}>
+            Teams and Access Settings
+          </Text.Title>
+          <TeamsSettings />
         </div>
-        <ApiTokenSettings />
+      )}
+      <Text.Title level={3} className={styles.title}>
+        API URL
+      </Text.Title>
+      <div>
+        <Field>
+          <Input value={apiUrlFromStatus} disabled />
```

Comment:
Real fix is passing value here from pluginStore, rest is refactoring from class to functional component